packages/nimble-components/src/banner/types.ts

```diff
@@ -11,3 +11,12 @@ export const BannerSeverity = {
 
 export type BannerSeverity =
     (typeof BannerSeverity)[keyof typeof BannerSeverity];
+
+/**
+ * The type of the detail associated with the `toggle`
+ * event on the banner.
+ */
+export interface BannerToggleEventDetail {
```

Comment:
Could just have made this minor change in the other PR to avoid the bubble up (just want to avoid massive changes to components and angular at the same time)

Reply:
I figured that would break the `nimble-angular` build, since it would still be depending on a version of `nimble-components` that didn't yet export the type. Don't we need the new version of `nimble-components` to exist and then bump `nimble-angular`'s dependency version before we reference the type from `nimble-angular`?